### ISSUE TITLE
passing in NULL to go-python function?

### ISSUE THREAD

**Issue by reiinakano** [sbinet/go-python#61]
I'm trying to use `PyList_SetSlice` and according to the documentation, we can set `itemlist` to NULL. How would I do this? I tried `C.NULL`, `nil`, and even 0 but nothing seems to work.

**Comment by sbinet** [comment]
hi @reiinakano 

could you please give a small compilable example of what you've tried?

**Comment by reiinakano** [comment]
Only this works
```
// origList is my original list from some other function
emptyList := python.PyList_New(0)
// empty original list
err := python.PyList_SetSlice(origList, 0, length, emptyList)
```
These others don't
```
// origList is my original list from some other function
emptyList := python.PyList_New(0)
// empty original list
err := python.PyList_SetSlice(origList, 0, length, nil)
```

```
// origList is my original list from some other function
emptyList := python.PyList_New(0)
// empty original list
err := python.PyList_SetSlice(origList, 0, length, 0)
```

```
import "C"
// origList is my original list from some other function
emptyList := python.PyList_New(0)
// empty original list
err := python.PyList_SetSlice(origList, 0, length, C.NULL)
```

**Comment by reiinakano** [comment]
Sorry, you asked for compilable. Here

```go
package main

import (
	"github.com/sbinet/go-python"
	"fmt"
)

import "C"

func main() {
	python.Initialize()

	origList := python.PyList_New(0)
	python.PyList_Append(origList, python.PyString_FromString("zdg"))
	fmt.Println(python.PyString_AsString(origList.Str()))

	emptyList := python.PyList_New(0)
	python.PyList_SetSlice(origList, 0, 1, emptyList)
	fmt.Println(python.PyString_AsString(origList.Str()))

	origList = python.PyList_New(0)
	python.PyList_Append(origList, python.PyString_FromString("zdg"))
	fmt.Println(python.PyString_AsString(origList.Str()))

	python.PyList_SetSlice(origList, 0, 1, nil)
	fmt.Println(python.PyString_AsString(origList.Str()))
}

```

**Comment by sbinet** [comment]
it's fixed with 2665c25.

thanks for your patience.

**Comment by reiinakano** [comment]
Thank you!
